json2html/jsonconv.py

```diff
@@ -23,6 +23,12 @@ def convert(self,**args):
 		'''
 		convert json Object to HTML Table format
 		'''
+		global table_attributes
+		table_attributes = ''
+		if 'table_attributes' in args:
+			table_attributes = args['table_attributes']
+		else:
+			table_attributes = "<table border=\"1\">"
```

Comment:
damn. sorry, on Japan time and it got late here, seems like I’ve potatoes on my eyes ;). amazingly it even rendered correctly in my browser.


On 02 Feb 2015, at 00:56, Varun Malhotra <notifications@github.com> wrote:

> In json2html/jsonconv.py:
> 
> > @@ -23,6 +23,12 @@ def convert(self,**args):
> >  		'''
> >  		convert json Object to HTML Table format
> >  		'''
> > +		global table_attributes
> > +		table_attributes = ''
> > +		if 'table_attributes' in args:
> > +			table_attributes = args['table_attributes']
> > +		else:
> > +			table_attributes = "<table border=\"1\">"
> Please check once, this will make the tag <table <table border="1">> as you have used table_init_markup = "<table %s>" %(table_attributes)
> a=a+ table_init_markup
> 
> Please replace this with table_attributes = 'border="1"'
> 
> —
> Reply to this email directly or view it on GitHub.
>